README.md

```diff
@@ -14,6 +14,12 @@ This will spin up the database via `docker` and launch the local API gateway.
 
 In parallel, it will use [Vite](https://vitejs.dev) to serve your app on [http://localhost:3000](http://localhost:3000). Vite will automatically update the browser via hot module reloading on save while providing extremely fast rebuilds.
 
+For the parallel execution, you will need to have `npm-run-all` installed:
+
+```shell
+npm install -g npm-run-all
+```
+
```

Comment:
As it is installed locally, that should not be necessary. I wonder why it did not work for you?

Reply:
You are right,  it is included locally, thus, it should work. I removed my global installation and tested it successfully. Thus, I will remove this line.